src/Chapter1.hs

```diff
@@ -632,8 +628,22 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = 
+  let posN = abs n
+  in let last1 = mod posN 10
+         last2
+           | posN > 9 = mod (div posN 10) 10 
+           | otherwise = 0
```

Comment:
`mod (div posN 10) 10` will return 0 in case of `<= 9`, so no need to check it separately 🙂 

Also, you can see that you use both:
```haskell
mod posN 10
div posN 10
```

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(a, b) = divMod posN 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Great suggestion! Refactored this function with `divMod`.